src/Discord.Net.WebSocket/DiscordSocketClient.cs

```diff
@@ -1464,6 +1464,12 @@ public override RestVoiceRegion GetVoiceRegion(string id)
 
                                     var data = (payload as JToken).ToObject<VoiceServerUpdateEvent>(_serializer);
                                     var guild = State.GetGuild(data.GuildId);
+                                    var cacheable = new Cacheable<IGuild, ulong>(guild, data.GuildId, guild != null,
+                                        async () => (IGuild) await ApiClient.GetGuildAsync(data.GuildId));
```

Comment:
Was the cast necessary? Not sure if the compiler would actually complain about that or not, but my first instinct just from reading is that it shouldn't.

Reply:
It did https://i.serux.pro/6c425faf71.png